.github/workflows/build-dev.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         run: cargo install dioxus-cli@0.5.0
 
       - name: Build
-        run: dx build --release
+        run: dx build --release --features dev
```

Comment:
Here you use `dev` but everywhere else `staging`, I think `dev` is better so the code needs some updates